release_notes/ocp_3_3_release_notes.adoc

```diff
@@ -151,10 +151,9 @@ parameter] to define the name of the GCS bucket to store objects in.
 [[ocp-support-docker-distribution-2-4]]
 ==== Support for docker distribution 2.4
 
-The {product-title} 3.3 registry provides support for docker distribution
-registry 2.4, and the features will be backported to {product-title} 3.2.
-Version 2.4 of the registry includes a variety of performance and usability
-enhancements, notably:
+The {product-title} 3.3 registry is based on docker distribution registry 2.4,
```

Comment:
Thinking now that we should init cap this (and the section title) as "Docker Distribution", no? From looking around at capitalization in https://github.com/docker/distribution.